test/controllers/user.e2e-spec.ts

```diff
@@ -73,6 +77,46 @@ describe('Users Controller', () => {
     });
   });
 
+  describe('POST /api/users/login-user', () => {
+    it('should return an error if fields are empty', async () => {
+      const response = await request(app.getHttpServer())
+        .post('/api/users/login-user')
+        .send({});
+
+      expect(response.statusCode).toBe(400);
+      expect(response.body.message.message).toStrictEqual([
+        'email must be an email',
+        'password must be a string',
+      ]);
+    });
+
+    it('should return an error if the email does not exist', async () => {
+      const response = await request(app.getHttpServer())
+        .post('/api/users/login-user')
+        .send({
+          email: 'fakeEmail@email.io',
+          password: 'password',
+        });
+
+      expect(response.body.message.message).toStrictEqual(
+        'fakeEmail@email.io does not exist',
+      );
+    });
+
+    it('should verify the user and return a JWT token', async () => {
+      await createUser(app, { email: 'testaccount@email.io' });
+
+      const response = await request(app.getHttpServer())
+        .post('/api/users/login-user')
+        .send({
+          email: 'testaccount@email.io',
+          password: 'password',
+        });
+
+      expect(response.status).toBe(201);
```

Comment:
If this test is returning the JWT token, can we expect the `response.body` to be truthy as well? 🤔

Reply:
![image](https://user-images.githubusercontent.com/28055566/174074229-a5c796d4-f0ae-4240-88ca-3b232170cd21.png)

done